### ISSUE TITLE
[Validator] Broken sr_Latn validator translations (plurals)

### ISSUE THREAD

**Issue by knezmilos13** [symfony/symfony#35276]
validators.sr_Latn.xlf (Serbian, written with latin script) has wrong plurals for all translations that require them.

```
<trans-unit id="19">
    <source>This value is too long. It should have {{ limit }} character or less.|This value is too long. It should have {{ limit }} characters or less.</source>
    <target>Vrednost je predugačka. Trebalo bi da ima {{ limit }} karakter ili manje.|Vrednost je predugačka. Trebalo bi da ima {{ limit }} karaktera ili manje.</target>
</trans-unit>
```

Note how there are only two plural options there. However, validators.sr_Cyrl.xlf (Serbian, written with cyrilic script) has three plurals, and that would be the correct number and works properly:

```
<trans-unit id="19">
    <source>This value is too long. It should have {{ limit }} character or less.|This value is too long. It should have {{ limit }} characters or less.</source>
    <target>Вредност је предугачка. Треба да има {{ limit }} карактер или мање.|Вредност је предугачка. Треба да има {{ limit }} карактера или мање.|Вредност је предугачка. Треба да има {{ limit }} карактера или мање.</target>
</trans-unit>
```

It appears that this commit is to blame:
https://github.com/symfony/symfony/commit/9e9a57a5446d6855c0c0f4f343a5220f1dd7e9ee

The two plural forms can often be the same (depends on the word), so I guess the user that contributed the commit wanted to remove the duplication. Perhaps older versions of Symfony were less strict and just used what plurals were available? Or no one simply noticed the error so far:

![latinica](https://user-images.githubusercontent.com/2862958/72054199-ccdb6600-32c8-11ea-9e76-79ff012ca756.png)

I can fix the translations, but haven't contributed to Symfony so far, so I felt it was better to open an issue first. To which branch should I make a pull request? I've noticed this error only after switching from Symfony 3.4 to Symfony 4.4, and it seems to work in 3.4.40 (which I have in production).

**Comment by fabpot** [comment]
Translations are managed in a very specific way. Everything should happen in 3.4 first and then we merge into the other branches.
